docs/content/guides/author-apps/kubernetes/how-to-kubernetes-resource/index.md

```diff
@@ -0,0 +1,128 @@
+
+`kubeConfig` is leveraged by Radius for information needed to build connections with the user's remote Kubernetes clusters:
+
+{{< rad file="snippets/app-kubernetes.bicep" embed=true marker="//KUBERNETES" >}}
+
+## Step 2:  Add a Kubernetes secret resource
+
+Add a [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/) to your `app.bicep` file. This secret will contain a small amount of sensitive data such as a password, a token, or a key:
+
+{{< rad file="snippets/app-kubernetes.bicep" embed=true marker="//SECRET" >}}
+
+> Refer to the [Kubernetes overview page]({{< ref "/guides/author-apps/kubernetes/overview#resource-library" >}}) for additional information about available types.
+
+## Step 3: Add a container and use the secret you just defined
+
+Add a  Radius [container]({{< ref "guides/author-apps/containers" >}}) to your application:
+
+{{< rad file="snippets/app-kubernetes.bicep" embed=true marker="//APPLICATION" >}}
+
+## Step 4: Deploy your Radius Application
+
+Deploy your application with the `rad` CLI:
+
+```bash
+rad run ./app.bicep -a demo
+```
+
+Your console output should look similar to:
+
+```
+Building ./app.bicep...
+Deploying template './app.bicep' for application 'demo' and environment 'default' from workspace 'default'...
+
+Deployment In Progress... 
+
+...                  demo            Applications.Core/containers
+
+Deployment Complete
+
+Resources:
+    demo            Applications.Core/containers
+
+Starting log stream...
+```
+
+Navigate to the `Container Metadata` tab and the `Environment variables` section which are located near the bottom of the Radius demo webpage, there will be row dedicated to your Kubernetes secret object:
+
+{{< image src="demo-secret-object.png" alt="Screenshot of Radius Demo app `Environment variables section" width="700px" >}}
+
+You can also prove the deployed Kubernetes secret was created by using `kubectl`, and running the following command with your specific secret name:
+
+```bash
+kubectl describe secret my-secret  -n default-demo
+```
+
+Your console output should contain the following section:
+
+```
+Name:         my-secret
+Namespace:    default-demo
+Labels:       <none>
+Annotations:  <none>
+
+Type:  Opaque
+
+Data
+====
+key:            15 bytes
+my-secret-key:  15 bytes
+```
+
+## Cleanup
+
+To delete your Radius specific Kubernetes resources you'll need to run:
+
+```bash
+rad app delete -a demo
+```
+
+Your console output should look similar to:
+
+```
+Application demo deleted
+```
+
+Once your Radius Application has been deleted you can delete the remaining Kubernetes resources you created by running:
+
+```bash
+kubectl delete all --all -n default-demo
+```
+
+Your console output should look similar to:
+
+```
+pod "demo-74794c5b55-jw4db" deleted
+service "demo" deleted
+deployment.apps "demo" deleted
+replicaset.apps "demo-74794c5b55" deleted
+```
```

Comment:
This doesn't look like the correct output. Could you update this step to just delete the secret instead of all the resources inside of the default-demo namespace?

Reply:
@AaronCrawfis Hey funny enough I just reran it and it seems like `rad app delete -a demo` deletes all the resources in the namespace `default-demo` where the secret is living so not sure we even need this step.

Reply:
Is there a specific resource or namespace the user should be deleting?

Reply:
If you've deleted your application, then there should only be the secret remaining in the namespace afterwards. If you're seeing the pod/service/deployment/replicaset all being deleted, then the application deletion didn't work correctly.

Reply:
When you have time I'd love to do a demo on this and figure out what's missing